pinpoint-hbase/docker-compose.yml

```diff
@@ -15,20 +15,28 @@ services:
       - /home/pinpoint/hbase
       - /home/pinpoint/zookeeper
     expose:
-#      # zookeeper
-#      - "2181"
       # HBase Master API port
       - "60000"
```

Comment:
how about also changing 60000 to 16000 and all the code below that is related (https://hbase.apache.org/book.html#hbase_default_configurations)

Reply:
yes, I think that is more related to the others.